api/v1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
so it go1.17 change, I do not have a problem moving forward to 1.17, but it will be under a separate file and include changes to our tools image https://github.com/openshift-kni/performance-addon-operators/blob/c1dfe722119075fae8368ad05f0df85a733f0b91/openshift-ci/Dockerfile.tools#L5

Reply:
Yes, better to postpone the move to golang 1.17. @marioferh please switch back to golang 1.16.z for the time being